config/webpack/webpack.config.custom.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+
+module.exports = {
+  resolve: {
+    alias: {
+      '@root': path.resolve(__dirname, '../..'),
+      '@app': path.resolve(__dirname, '../../app'),
+      '@javascript': path.resolve(__dirname, '../../app/javascript'),
+      '@lib': path.resolve(__dirname, '../../app/javascript/lib'),
+      '@components': path.resolve(__dirname, '../../app/javascript/components'),
+      '@widgets': path.resolve(
+        __dirname,
+        '../../app/javascript/components/widgets',
+      ),
+      '@messages': path.resolve(
+        __dirname,
+        '../../app/javascript/components/widgets/messages',
+      ),
+      '@numbers': path.resolve(
+        __dirname,
+        '../../app/javascript/components/widgets/numbers',
+      ),
+      '@weather': path.resolve(
```

Comment:
I think these aliases are a good balance of general (`@components`) and specific (`@weather`). 👍